src/main/kotlin/racingcar/CarManager.kt

```diff
@@ -0,0 +1,13 @@
+package racingcar
+
+class CarManager {
+    fun separateCarNames(carNames: String): List<String> = carNames.split(",")
+
+    fun makeCarModelList(carNames: List<String>): List<Car> {
+        var modelList = mutableListOf<Car>()
+        carNames.forEach {
+            modelList.add(Car(it))
+        }
```

Comment:
forEach 보단 이렇게 줄이면 더 좋을 것 같아요
```suggestion
        var modelList = carNames.map {Car(it)}
```

Reply:
헐 이런방식도 있었네요 좋은 리뷰 감사합니다!!